changelogs/CHANGELOG-1.4.md

```diff
@@ -1,5 +1,307 @@
 # Changelog since v1.3.0
```

Comment:
Shouldn't this be Changelog since 1.4.0?

Reply:
1.4.0 changelog describes all changes 1.3.0, it's how CHANGELOG-1.1 and CHANGELOG-1.2 were written.